dojo/templates/base.html

```diff
@@ -3,6 +3,7 @@
 {% load authorization_tags %}
 {% load i18n %}
 {% load static %}
+{% load problem_tags %}
```

Comment:
How about we move the `check_problem_enabled` tag into one of the existing files? Maybe `display_tags` or `navigation_tags`?